code/game/objects/items/bodybag.dm

```diff
@@ -12,6 +12,28 @@
 	R.add_fingerprint(user)
 	qdel(src)
 
+/obj/item/bodybag/medical_dnr
+	name = "body bag (medical DNR)"
+	desc = "A folded bag designed for the storage and transportation of cadavers under a Medical DNR Notice."
+	icon = 'icons/obj/bodybag.dmi'
+	icon_state = "MMTODO"
+
+/obj/item/bodybag/medical_dnr/attack_self(mob/user)
+	var/obj/structure/closet/body_bag/medical_dnr/R = new /obj/structure/closet/body_bag/medical_dnr(user.loc)
```

Comment:
This should be a var on `/obj/item/bodybag` so that attack_self doesn't have to be copied 3 times

Reply:
yeahhhhh I copy/pasted for proof of concept - Ill avoid coding warcrimes once i do a pass over with the sprites coming up :xdd: